src/systems/camera_video_recorder/CameraVideoRecorder.hh

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef IGNITION_GAZEBO_CAMERAVIDEORECORDER_SYSTEM_HH_
+#define IGNITION_GAZEBO_CAMERAVIDEORECORDER_SYSTEM_HH_
+
+#include <memory>
+#include <ignition/gazebo/config.hh>
+#include <ignition/gazebo/Export.hh>
+#include <ignition/gazebo/System.hh>
+
+namespace ignition
+{
+namespace gazebo
+{
+// Inline bracket to help doxygen filtering.
+inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace systems
+{
+  class CameraVideoRecorderPrivate;
+
+  /** \class CameraVideoRecorder CameraVideoRecorder.hh \
+   * ignition/gazebo/systems/CameraVideoRecorder.hh
+  **/
+  /// \brief Record video from a camera sensor
+  /// The system takes in the following parameter:
+  ///   <topic>    Name of topic for the video recorder service. If this is
```

Comment:
Should we call this `<service>` or `<service_topic>`, instead of `<topic>`?

Reply:
renamed to `<service>` d90f78e